ui/src/components/charts/GrowthChart.js

```diff
@@ -77,26 +77,25 @@ const GrowthChart = ({ sqon, setSQON }) => (
                   animate={false}
                   tooltip={({ id, value, label }) => ChartTooltip({ value, label })}
                   theme={theme.chart}
-                  onClick={
-                    (data) => ({})
-                    // setSQON(
-                    //   addInSQON(
-                    //     {
-                    //       op: 'and',
-                    //       content: [
-                    //         {
-                    //           op: 'in',
-                    //           content: {
-                    //             field: 'type',
-                    //             value: (data.keys || []).sort(),
-                    //           },
-                    //         },
-                    //       ],
-                    //     },
-                    //     sqon,
-                    //   ),
-                    // )
-                  }
+                  onClick={(data) => {
+                    setSQON(
+                      addInSQON(
+                        {
+                          op: 'and',
+                          content: [
+                            {
+                              op: 'in',
+                              content: {
+                                field: 'type',
```

Comment:
`field` is updated to `fieldName` in https://github.com/nci-hcmi-catalog/portal/pull/1101/files